Update Heroku references to Glitch (#5590)
stripe/stripe-android: example/README.md

### Install
1. Clone the `stripe-android` repository.
2. Open the project in Android Studio.

> 3. After [remixing the Glitch project](#remix-the-example-project-on-glitch) and [configuring the app](#configure-the-app), build and run the project.

Reviewer: This is now an optional step, as people can just use our example Glitch, right?

Author: No, they don't have our publishable key. In the future we want to return it in the backend call, like we do in the paymentsheet-example app. For now this is just migrating off Heroku.

<img width="215" height="108" src="https://raw.githubusercontent.com/stripe/stripe-android/master/example/images/run.png" />

### Remix the example project on Glitch

> We provide an example backend hosted on Glitch, allowing you to easily test an integration end-to-end.

Reviewer: ```suggestion
We provide an example backend hosted on Glitch, allowing you to easily test an integration end-to-end. You can also [remix](https://glitch.happyfox.com/kb/article/23-remix/) it to make it your own.
```

Author: They need to remix, so I wouldn't say `You can also remix`

1. [Open the Glitch project](https://glitch.com/edit/#!/stripe-example-mobile-backend).
2. Click on "Remix", on the top right.

> 3. In your newly created project, open the `.env` file, listed on the left panel.

Reviewer: ```suggestion
3. In your newly created project, open the `.env` file in the left sidebar.
```

4. Set your [Stripe testmode secret key](https://dashboard.stripe.com/test/apikeys) as the `STRIPE_TEST_SECRET_KEY` field.

> 5. Your backend implementation should now be running. You can see the logs by clicking on "Logs" at the bottom bar.

Reviewer: ```suggestion
5. Your backend implementation should now be running. You can see the logs by clicking on "Logs" in the bottom bar.
```
